test/edu-benefits/1995/00-required.e2e.spec.js

```diff
@@ -0,0 +1,102 @@
+if (process.env.BUILDTYPE !== 'production') {
+  const E2eHelpers = require('../../util/e2e-helpers');
+  const Timeouts = require('../../util/timeouts.js');
+  const EduHelpers = require('../../util/edu-1995-helpers');
+
+  module.exports = E2eHelpers.createE2eTest(
+    (client) => {
+      EduHelpers.initApplicationSubmitMock();
+
+      // Ensure introduction page renders.
+      client
+        .url(`${E2eHelpers.baseUrl}/education/apply-for-education-benefits/application/1995`)
+        .waitForElementVisible('body', Timeouts.normal)
+        .assert.title('Apply for education benefits: Vets.gov')
+        .waitForElementVisible('div.form-progress-buttons', Timeouts.slow)
+        .click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.overrideVetsGovApi(client);
+      E2eHelpers.overrideSmoothScrolling(client);
+      E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+      // Veteran information page.
+      client
+        .waitForElementVisible('input[name="root_veteranFullName_first"]', Timeouts.slow);
+      EduHelpers.completeVeteranInformation(client, EduHelpers.testData.veteranInformation.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/veteran-information');
+
+      // Benefits eligibility
+      client
+        .waitForElementVisible('label[for="root_benefit"]', Timeouts.slow);
+      EduHelpers.completeBenefitsSelection(client, EduHelpers.testData.benefitSelection.data);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/benefits-elibility/benefits-selection');
+
+      // Service periods page.
+      client
+        .waitForElementVisible('label[for="root_view:newService"]', Timeouts.slow);
+      EduHelpers.completeServicePeriods(client, EduHelpers.testData.servicePeriods.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/military-history/service-periods');
+
+      // Military service page.
+      client
+        .waitForElementVisible('label[for="root_view:hasServiceBefore1978"]', Timeouts.slow);
+      // Another mysteriously required pause. If we don't wait here, then the click below will
```

Comment:
I'm not sure why this is needed, but when profiling the navigation to this page, I found it has a lot of unnecessary DOM updates to the `YesNoWidget`, which might be the cause. It still renders super quickly (6 - 7 ms), though.

![image](https://cloud.githubusercontent.com/assets/12970166/23319221/8aa3a9d8-fa8a-11e6-9514-c0f5bced42e4.png)